automation_tools/__init__.py

```diff
@@ -1186,11 +1185,22 @@ def ak_install(admin_password=None, clean_beaker=True,
         '--activationkey="{1}"'.format(org, act_key)
     )
     run(cmd)
+
+    # Refresh subscriptions and clean up YUM
     print('Refreshing Subscription-manager.')
     run('subscription-manager refresh')
     print('Performing yum clean up.')
     run('yum clean all', quiet=True)
 
+    # Set yum stdout log level to be less verbose
```

Comment:
Just saw @lpramuk comment. I think the changes could be request before merging, imo needing to rush is not an excuse to have code without quality but have a fast peace on updating the PR to get it merged.